catalog/spatial/registry/registry-federation-admin-service-impl/src/test/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitializationTest.java

```diff
@@ -155,55 +150,64 @@ public void initWithPreviousEntryWithNameChange() throws Exception {
   @Test
   public void initWithGetLocalRegistryNodeException() throws Exception {
     doThrow(FederationAdminException.class)
+        .doReturn(Optional.empty())
         .when(federationAdminService)
         .getLocalRegistryIdentityMetacard();
     identityNodeInitialization.init();
-    verify(executorService).schedule(any(Runnable.class), anyLong(), any(TimeUnit.class));
+    verify(identityNodeInitialization, times(2)).updateOrCreateIdentity();
   }
 
   @Test
+  public void initCreateTimeout() throws Exception {
+    try {
```

Comment:
❓ Do we need to change the retry interval for the test so we are not waiting the full retry amount multiple times for this test?

Reply:
The retry time is 10 milliseconds (total of 30milliseconds) so i didn't think it would be an issue.